### PR TITLE
add configuration for cuda

### PR DESCRIPTION
**Pull request by muleisheng**

### CODE REVIEW

File: modules/perception/common/cuda_conf.cc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+#include "modules/perception/common/cuda_conf.h"
+
+
+namespace apollo {
+namespace perception {
+
+CudaConfig::CudaConfig(): block_width_(32)
+    , block_height_(32) {
+}
```
Comment (jinghaomiao) [inline modules/perception/common/cuda_conf.cc]: Use const static variable for 32.

File: modules/perception/common/cuda_conf.cc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+CudaConfig& CudaConfig::instance() {
+    static CudaConfig inst;
+    return inst;
+}
+
+};  // end namespace perception
+};  // end namespace apollo
+
```
Comment (jinghaomiao) [inline modules/perception/common/cuda_conf.cc]: Add a unit test.

Comment (muleisheng) [reply modules/perception/common/cuda_conf.cc]: OK, I will add test case for cuda_conf

File: modules/perception/common/cuda_conf.cc (hunk 1 above)
Comment (muleisheng) [reply modules/perception/common/cuda_conf.cc]: OK, I will modify this issue. Thanks for your proposal.

**Comment by fusiled** [comment]
`wget`-ting the URL  `http://www.baiduapollo.club/apollo-docker/jdk-8u144-linux-arm64-vfp-hflt.tar.gz`
 inside the script  `docker/build/installers/install_bazel_aarch64.sh` returns code 403 (at least from Europe). If you try to visit the webpage you gat a message related to the ICP Filing. It is not clear if the issue is related to the website.

**Comment by fusiled** [comment]
> `wget`-ting the URL `http://www.baiduapollo.club/apollo-docker/jdk-8u144-linux-arm64-vfp-hflt.tar.gz`
> inside the script `docker/build/installers/install_bazel_aarch64.sh` returns code 403 (at least from Europe). If you try to visit the webpage you gat a message related to the ICP Filing. It is not clear if the issue is related to the website.

Actually, using the IP address from commit 
[408e103](https://github.com/ApolloAuto/apollo/pull/5953/commits/408e103dc93fc1a604fcfc9fbc61c150cf4931b2)  is accessible. A restore of this commit should fix the problem.

**Comment by zhxt** [comment]
@fusiled Looks like that domain has not been completed the domain name registration procedure(local law). It will take about three weeks(or more) after the request is submitted .

**Comment by gouchaoer** [comment]
@jinghaomiao hey, what's wrong with this PR?
I have been waiting this PR the be merged. Because I need to run apollo on Xavier.
Do you have any idea?

**Pull request closed without merging**